plugins/pdr_deterministic_plugin/ufm_sim_web_service/isolation_mgr.py

```diff
@@ -458,7 +445,13 @@ def get_isolation_state(self):
     def start_telemetry_session(self):
         self.logger.info("Starting telemetry session")
         guids = self.get_requested_guids()
-        response = self.ufm_client.start_dynamic_session(Constants.PDR_DYNAMIC_NAME, self.telemetry_counters, self.t_isolate, guids)
+        # bring telemetry data on disabled ports
+        extra_configuration = {
```

Comment:
As it is not dynamic configuration - may be it could be defined somewhere in upper stage?

Reply:
done